ghost/core/core/server/data/migrations/versions/5.24/2022-11-21-15-03-populate-source-column-with-html-for-emails.js

```diff
@@ -0,0 +1,16 @@
+const logging = require('@tryghost/logging');
+const {createTransactionalMigration} = require('../../utils');
+
+module.exports = createTransactionalMigration(
+    async function up(knex) {
+        logging.info('Populating source from html in emails table');
+        await knex('emails')
```

Comment:
Can you get the number of affected rows returned from this `.update` call, and log it out for visibility? 🙂

Reply:
Done, thanks